test/uk/gov/hmrc/merchandiseinbaggage/connectors/CurrencyConversionConnectorSpec.scala

```diff
@@ -24,9 +24,10 @@ import java.time.LocalDate
 
```

Comment:
Please revert these changes as `CurrencyConversionStub` doesn't need to a class, thanks.